src/index.html

```diff
@@ -9,6 +9,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          class="logo" 
+          src="images/logo.png" 
+          alt="logo">
+      </a>
+      <nav class="navbar">
+        <ul>
+          <li><a class="is-active" href="#">Apple</a></li>
```

Comment:
it would be better to add classes and use them in CSS (not tags)
```suggestion
          <li class="nav-item">
            <a class="nav-item__link is-active" href="#">Apple</a>
          </li>
```